apps/balancer-tools/src/app/apr/pool/(components)/PoolTokens.tsx

```diff
@@ -1,27 +1,32 @@
 import { PlotTitle } from "#/components/Plot";
 import { fetcher } from "#/utils/fetcher";
 
+import { formatDateToMMDDYYYY } from "../../api/(utils)/date";
 import { BASE_URL, PoolStatsResults } from "../../api/route";
 import PoolTokensTable from "./PoolTokensTable";
 
 export default async function PoolTokens({
-  roundId,
   poolId,
+  startAt,
+  endAt,
 }: {
-  roundId?: string;
   poolId: string;
+  startAt: Date;
+  endAt: Date;
 }) {
   const poolData: PoolStatsResults = await fetcher(
     `${BASE_URL}/apr/api/?poolId=${poolId}${
-      roundId ? `&roundId=${roundId}` : ""
-    }`,
+      startAt ? `&startAt=${formatDateToMMDDYYYY(startAt)}` : ""
+    }${endAt ? `&endAt=${formatDateToMMDDYYYY(endAt)}` : ""}`,
```

Comment:
we use this URL so much I think we should aim to have a function to avoid repeating it everywhere 😅